geopandas/tests/test_geodataframe.py

```diff
@@ -502,7 +502,9 @@ def test_geodataframe_geojson_bbox(self):
     def test_pickle(self):
         import pickle
         df2 = pickle.loads(pickle.dumps(self.df))
-        assert str(df) == str(df2)
+        assert self.df._geometry_array.equals(df2._geometry_array).all()
```

Comment:
Should this always be true? 
Because I can imagine that if the the wkb is deserialized, new GEOSGeometry objects will be created which have different integers pointers

Reply:
ah, no, ignore, this the `equals` method of the GeometryArray, which of course takes care of that :-)